src/NServiceBus.Transport.Msmq.Tests/MsmqUtilitiesTests.cs

```diff
@@ -61,6 +61,54 @@ public void Should_fetch_the_replyToAddress_from_responsequeue_for_backwards_com
         }
 
         [Test]
+        public void Should_not_override_replyToAddress()
+        {
+            var message = MsmqUtilities.Convert(
+                new OutgoingMessage("message id", new Dictionary<string, string>
+                    {
+                        {Headers.ReplyToAddress, "SomeAddress"}
+                    }, new byte[0]),
+                new List<DeliveryConstraint>());
+
+            message.ResponseQueue = new MessageQueue(new MsmqAddress("local", RuntimeEnvironment.MachineName).FullPath);
+            var headers = MsmqUtilities.ExtractHeaders(message);
+
+            Assert.AreNotEqual("local@" + RuntimeEnvironment.MachineName, headers[Headers.ReplyToAddress]);
```

Comment:
This seems redundant? (the next assert covers this?)

Reply:
yeah, I'll remove the redundant assert.